### PR TITLE
Fix part of #1287: Fix lint issues in app module in the kotlin files (administratorcontrols)

### PR DESCRIPTION
**Pull request by anandwana001**
## Explanation

Fixes part of #1287
`administratorcontrols` folder `ktlint` check

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [ ] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsActivity.kt [hunk 1]
```diff
@@ -12,8 +12,12 @@ import org.oppia.app.settings.profile.ProfileListActivity
 import javax.inject.Inject
 
 /** Activity for Administrator Controls. */
-class AdministratorControlsActivity : InjectableAppCompatActivity(), RouteToProfileListListener, RouteToAppVersionListener {
-  @Inject lateinit var administratorControlsActivityPresenter: AdministratorControlsActivityPresenter
+class AdministratorControlsActivity :
+  InjectableAppCompatActivity(),
+  RouteToProfileListListener,
+  RouteToAppVersionListener {
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsActivity.kt:18]: Is this the default behaviour of ktlint?
Meaning I was expecting it to be like this:
```
class AdministratorControlsActivity : InjectableAppCompatActivity(),
  RouteToProfileListListener,
  RouteToAppVersionListener {
```

File: app/src/main/java/org/oppia/app/administratorcontrols/appversion/AppVersionViewModel.kt [hunk 2]
```diff
@@ -19,7 +19,10 @@ class AppVersionViewModel @Inject constructor(
   val versionName = ObservableField<String>(BuildConfig.VERSION_NAME)
 
   private val lastUpdateDateTime =
-    fragment.activity!!.packageManager.getPackageInfo(fragment.activity!!.packageName, /* flags= */ 0).lastUpdateTime
+    fragment.activity!!.packageManager.getPackageInfo(
+      fragment.activity!!.packageName, /* flags= */
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/administratorcontrols/appversion/AppVersionViewModel.kt]: Nit Change:
```
    fragment.activity!!.packageManager.getPackageInfo(
      fragment.activity!!.packageName,
       /* flags= */ 0
    ).lastUpdateTime
```

File: app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsViewModel.kt [hunk 3]
```diff
@@ -32,27 +32,42 @@ class AdministratorControlsViewModel @Inject constructor(
...
     val itemViewModelList: MutableList<AdministratorControlsItemViewModel> = mutableListOf(
       AdministratorControlsGeneralViewModel()
     )
     itemViewModelList.add(AdministratorControlsProfileViewModel(routeToProfileListListener))
-    itemViewModelList.add(AdministratorControlsDownloadPermissionsViewModel(fragment, logger, profileManagementController,
-      userProfileId, deviceSettings))
+    itemViewModelList.add(
+      AdministratorControlsDownloadPermissionsViewModel(
+        fragment, logger, profileManagementController,
+        userProfileId, deviceSettings
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsViewModel.kt]: Is this correct or should this be in multiple lines like:
```
        fragment,
        logger,
        profileManagementController,
        userProfileId,
        deviceSettings
```

**Review by rt4914 — CHANGES_REQUESTED** [review]
Thanks. PTAL

File: app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsActivity.kt (hunk 1 above)
Comment (anandwana001) [reply app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsActivity.kt:18]: Ktlint says there should be a new line after `:` and if there are multiple things there should be new line after `,`

File: app/src/main/java/org/oppia/app/administratorcontrols/appversion/AppVersionViewModel.kt (hunk 2 above)
Comment (anandwana001) [reply app/src/main/java/org/oppia/app/administratorcontrols/appversion/AppVersionViewModel.kt]: ok, this should be on same line, I will update

File: app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsViewModel.kt (hunk 3 above)
Comment (anandwana001) [reply app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsViewModel.kt]: both ways are correct.

File: app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsActivity.kt (hunk 1 above)
Comment (rt4914) [reply app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsActivity.kt:18]: Okay than in that case, it should be applied to https://github.com/oppia/oppia-android/pull/1291#discussion_r439205087 for better consistency.

File: app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsViewModel.kt (hunk 3 above)
Comment (rt4914) [reply app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsViewModel.kt]: Okay, then in that case, keep it in multiple lines for better code clarity and consistency with https://github.com/oppia/oppia-android/pull/1291#discussion_r439205779

File: app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsActivity.kt (hunk 1 above)
Comment (anandwana001) [reply app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsActivity.kt:18]: yeah, definitely

File: app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsViewModel.kt (hunk 3 above)
Comment (anandwana001) [reply app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsViewModel.kt]: ok

File: app/src/main/java/org/oppia/app/administratorcontrols/appversion/AppVersionViewModel.kt [hunk 4]
```diff
@@ -19,7 +19,10 @@ class AppVersionViewModel @Inject constructor(
   val versionName = ObservableField<String>(BuildConfig.VERSION_NAME)
 
   private val lastUpdateDateTime =
-    fragment.activity!!.packageManager.getPackageInfo(fragment.activity!!.packageName, /* flags= */ 0).lastUpdateTime
+    fragment.activity!!.packageManager.getPackageInfo(
+      fragment.activity!!.packageName,
+      /* flags= */0
```
Comment (rt4914) [inline app/src/main/java/org/oppia/app/administratorcontrols/appversion/AppVersionViewModel.kt]: Spacing issue: there should be space before `0`
`/* flags= */ 0`

**Review by rt4914 — CHANGES_REQUESTED** [review]
LGTM, one final nit suggested.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

File: app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityPresenter.kt [hunk 5]
```diff
@@ -26,18 +28,29 @@ class AdministratorControlsActivityPresenter @Inject constructor(private val act
...
+      .findFragmentById(
+        R.id.administrator_controls_activity_fragment_navigation_drawer
+      ) as NavigationDrawerFragment
     navigationDrawerFragment.setUpDrawer(
-      activity.findViewById<View>(R.id.administrator_controls_activity_drawer_layout) as DrawerLayout,
+      activity
+        .findViewById<View>(
+          R.id.administrator_controls_activity_drawer_layout
+        ) as DrawerLayout,
       toolbar, /* menuItemId = */ 0
```
Comment (veena14cs) [inline app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityPresenter.kt]: Make it `/* menuItemId= */`

**Review by veena14cs — APPROVED** [review]
LGTM beside one nit change.

File: app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityPresenter.kt (hunk 5 above)
Comment (anandwana001) [reply app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityPresenter.kt]: Done

**Comment by anandwana001** [comment]
> LGTM beside one nit change.

Thanks for the review :)

File: app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityPresenter.kt (hunk 5 above)
Comment (rt4914) [reply app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityPresenter.kt]: Resolving this on behalf of @veena14cs

**Pull request merged**